x/ratelimit/keeper/quota.go

```diff
@@ -0,0 +1,65 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/Stride-Labs/stride/v4/x/ratelimit/types"
+)
+
+// Add new quota to the store
+func (k Keeper) AddQuota(ctx sdk.Context, quota types.Quota) {
```

Comment:
My apologies, I had this wrong in the spec, this should probably be `SetQuota` instead of `AddQuota`